pages/Configuring/Monitors.md

```diff
@@ -53,6 +53,25 @@ monitor=DP-1, 1920x1080, 0x0, 1
 monitor=DP-2, 1920x1080, -1920x0, 1
 ```
 
+Recall the `position` is calculated from the top-left corner. Thus, a larger number
+places a monitor lower relative to the others.
+
```

Comment:
The second sentence doesn't really make sense here. Anything with `-` in front will go above, and anything without will go below. It's an inverted Y cartesian coordinate system.

Reply:
I'll rephrase to improve the wording and resubmit. I spent a good bit of time confused about the inverted nature of the y-axis and think an example would be beneficial for the documentation.